h2o-extensions/xgboost/build.gradle

```diff
@@ -27,6 +28,7 @@ dependencies {
     compile "ai.h2o:xgboost4j-linux-minimal:${xgboost4jVersion}"
     compile "ai.h2o:xgboost4j-osx-minimal:${xgboost4jVersion}"
 
+    testCompile project(":h2o-app")
```

Comment:
This gives a cycle in dependencies (if we ignore that it is a dependency for test).